src/store/initial-state.js

```diff
@@ -167,18 +165,17 @@ const applyUrlParametersToNonPipelineState = (state, urlParams) => {
  */
 export const mergeLocalStorage = (state) => {
   const localStorageState = loadLocalStorage(localStorageName);
-  const localStorageRunsMetadataState = loadLocalStorage(
-    localStorageRunsMetadata
-  );
+
   Object.keys(localStorageState).forEach((key) => {
     if (!(key in state)) {
       delete localStorageState[key];
     }
   });
+
   const allLocalStorageState = {
     ...localStorageState,
-    ...{ runsMetadata: localStorageRunsMetadataState },
   };
+
   return deepmerge(state, allLocalStorageState);
```

Comment:
NIT: maybe since there is only one localStorage now, you can remove `allLocalStorageState` and pass `localStorageState` directly in the deepmerge?

Reply:
Done!